Card.cs

```diff
@@ -0,0 +1,77 @@
+﻿using EnumColors;
+using System;
+
+namespace Task1_GUI
+{
+    internal class Card : IComparable
+    {
+        public int Nominal { get; set; }
```

Comment:
I wouldn't like to allow somebody set incorrect values as a Nominal or Color. You need to specify the `set` method or keep only `get`